editor/selectors.js

```diff
@@ -130,6 +131,31 @@ export function getEditedPostVisibility( state ) {
 }
 
 /**
+ * Return true if the post being edited has already been published.
+ *
+ * @param  {Object}   state Global application state
+ * @return {Boolearn}       Whether the post has been bublished
+ */
+export function isEditedPostAlreadyPublished( state ) {
```

Comment:
Question: Would this make more or less sense without "Already". I think it helps highlight that it tests whether the post was scheduled at a date before the current time, but maybe too much so to the extent that it might not be intuitive to use for a simple publish test.